app/components/datafruits-player.hbs

```diff
@@ -100,11 +100,12 @@
             type="range"
             min="0"
             max="1"
-            step="0.1"
+            step="0.01"
             aria-label={{t "player.aria.volume"}}
             {{on "input" this.volumeChanged}}
             value={{this.volume}}
           >
+          <span>{{this.volumeString}}</span>
```

Comment:
```suggestion
          <span class="hidden md:block">{{this.volumeString}}</span>
```

I don't mind showing the value I guess but maybe consider hiding it on mobile since we're pretty constrained for space on there anyway

Reply:
thats a good point. I tried this out and the volume bar got misaligned during the hover and the new label was positioned below it. using `inline-block` seems to work fine 😎